src/App.jsx

```diff
@@ -1,17 +1,19 @@
-import "./App.css";
+import './App.css';
 
-import {Routes, Route} from 'react-router-dom';
-import HomePage from "./pages/HomePage";
-import ProductListing from "./pages/ProductListing";
+import { Routes, Route } from 'react-router-dom';
+import HomePage from './pages/HomePage';
+import ProductListing from './pages/ProductListing';
+import CartPage from './pages/CartPage';
 function App() {
-  return (
-    <div className="App home-container flex-col">
-      <Routes>  
-        <Route path="/" element={<HomePage />} />
-        <Route path="/products" element={<ProductListing />} />s
-      </Routes>
-    </div>
-  );
+	return (
+		<div className="App">
+			<Routes>
```

Comment:
suggestion, instead of writing the navigation logic here, you can abstract it into a new file and then import it.

Reply:
I'll refactor the code, once I'm done with the basic functionality!